report-generator/src/main/java/eu/cqse/teamscale/report/testwise/model/FileCoverage.java

```diff
@@ -107,6 +107,11 @@ public boolean isEmpty() {
 		return coveredRanges.isEmpty();
 	}
 
+	@Override
```

Comment:
I don't like comparables that leave out part of the attributes. To me it's unexpected and s subtle source of bugs. I'd prefer a comparator. That way you can clarify what is being compared with the comparators name.